metagraph/src/graph/representation/succinct/dbg_succinct.cpp

```diff
@@ -516,6 +516,27 @@ ::call_source_nodes(const std::function<void(node_index)> &callback) const {
     });
 }
 
+std::shared_ptr<const bit_vector> DBGSuccinct
+::get_last() const {
+    return std::shared_ptr<const bit_vector>(&get_boss().get_last(), [](const bit_vector*) {
+        // Do not destruct BOSS's last with shared_ptr
+    });
```

Comment:
```suggestion
    return std::shared_ptr<const bit_vector>(std::shared_ptr<const bit_vector>{}, &get_boss().get_last());
```
No need to write a custom destructor, you can just use the aliasing contructor.

Reply:
Didn't know it's a thing, thanks for pointing out!